author/topics/document-format/meta-attributes.adoc

```diff
@@ -51,20 +51,6 @@ If you’re using one of the officially supported Metanorma flavors,
 consult your flavor’s author documentation as well.
 
 
-== Passing as command-line flags
```

Comment:
@opoudjis quick question about removing the section about passing document attributes from CLI:

* is it still supported (AFAIK the Asciidoctor toolchain & AsciiDoc document that ability explicitly), but we want to guide authors to not use it for any attributes? or
* is it not supported by Metanorma toolchain?

Reply:
It is supported by the asciidoctor CLI tool, but we want to discourage people from using that, because long term we will decouple Metanorma from asciidoctor. (I don't think that's going to be any time soon, but all mention of the asciidoctor CLI tool has been taken out of the READMEs.)

It is not supported by the metanorma tool.